application/PlatformPlatform.sln

```diff
@@ -14,6 +14,9 @@ EndProject
 Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Tests", "shared-kernel\Tests\Tests.csproj", "{B232D7BB-396A-46FE-B6D2-95041412C835}"
 EndProject
 Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "account-management", "account-management", "{EEA1463E-71F8-44E6-8055-787DAA5E62D2}"
+	ProjectSection(SolutionItems) = preProject
+		account-management\package.json = account-management\package.json
+	EndProjectSection
```

Comment:
not sure if we could move the workspace package.json up one level into the root of the application folder